completion_aggregator/settings/aws.py

```diff
@@ -9,7 +9,12 @@ def plugin_settings(settings):
     """
     Modify the provided settings object with settings specific to this plugin.
     """
-    settings.COMPLETION_AGGREGATOR_BLOCK_TYPES = settings.ENV_TOKENS.get(
+    settings.COMPLETION_AGGREGATOR_BLOCK_TYPES = set(settings.ENV_TOKENS.get(
         'COMPLETION_AGGREGATOR_BLOCK_TYPES',
-        settings.COMPLETION_AGGREGATOR_BLOCK_TYPES
+        settings.COMPLETION_AGGREGATOR_BLOCK_TYPES,
+    ))
+
+    settings.COMPLETION_AGGREGATOR_SYNCHRONOUS_AGGREGATION = settings.ENV_TOKENS.get(
```

Comment:
Typo:  The setting here is called `COMPLETION_AGGREGATOR_SYNCHRONOUS_AGGREGATION`, but `COMPLETION_AGGREGATOR_ASYNC_AGGREGATION` is used everywhere else.

I had to change the setting name to test synchronous aggregation.

Reply:
Also, this setting should be documented in the README.

Reply:
Added to readme.